examples/generic_connector_configs/README.md

```diff
@@ -420,125 +439,139 @@ API.
 
 ___
 
 ### Slack Web API
+
 This example can be used to interact with
 [Slack's Web API](https://api.slack.com/apis).
 
 The configuration file for the Slack Web API can be found at
 [slack_secretless.yml](./slack_secretless.yml).
 
+> This configuration was created on June 22, 2020. You can read about
+what changes have been made in the
+[Slack changelog](https://api.slack.com/changelog)
+
 #### How to use this connector
+
 * Edit the supplied configuration to get your Slack
-[OAuth token](https://api.slack.com/legacy/oauth#flow)
+  [OAuth token](https://api.slack.com/legacy/oauth#flow)
+
 * Run Secretless with the supplied configuration(s)
-* Query the Slack API using
-`http_proxy=localhost:9030 curl -d {data} <Slack Endpoint URL>`
-or `http_proxy=localhost:9040 curl -d {data} <Slack Endpoint URL>`
-depending on if your endpoint requires JSON or URL encoded requests
+
+* Query the Slack API
+  ```
+  http_proxy=localhost:9030 curl -d {data} {Slack Endpoint URL}
+  ```
+  ```
+  http_proxy=localhost:9040 curl -d {data} {Slack Endpoint URL}
+  ```
+  Your query depends on if your endpoint requires JSON or URL encoded requests.
 
 #### Example Usage
 <details>
   <summary><b>Example setup to try this out locally...</b></summary>
-  <ol>
-    <li>Get the Slack <a href="https://slack.com/help/articles/215770388-Create-and-regenerate-API-tokens">application's tokens</a></li>
-    <li>
-      Store the token from your request in your local credential manager so
-      that it may be retrieved in your <code>secretless.yml</code>
-    </li>
-    <li>Run Secretless locally</li>
-    <code>
-      ./dist/darwin/amd64/secretless-broker \
-      <br />
-      -f examples/generic_connector_configs/slack_secretless.yml
-    </code>
-    <li>On another terminal window, make a request to Slack using Secretless</li>
-    <code>http_proxy=localhost:9030 curl -X POST --data '{"channel":"C061EG9SL","text":"I hope the tour went well"}' slack.com/api/chat.postMessage</code>
-  </ol>
+
+  1. Get the Slack
+  [application's tokens](https://slack.com/help/articles/215770388-Create-and-regenerate-API-tokens)
+  1. Store the token from your request in your local credential manager so that
+  it may be retrieved in your `secretless.yml`
+  1. Run Secretless locally
+     ```
+       ./dist/darwin/amd64/secretless-broker \
+       -f examples/generic_connector_configs/slack_secretless.yml
+     ```
+  1. On another terminal window, make a request to Slack using Secretless
+     ```
+       http_proxy=localhost:9030 curl -X POST --data '{"channel":"C061EG9SL",
+       "text":"I hope the tour went well"}' slack.com/api/chat.postMessage
+     ```
+
 </details>
 
 ___
 
 ### Splunk API
-This example can be used to interact with [Splunk's API](https://api.slack.com/apis).
 
-The configuration file for the Splunk Web API can be found at [splunk_secretless.yml](./splunk_secretless.yml).
+This example can be used to interact with
+[Splunk's API](https://api.slack.com/apis).
```

Comment:
This link points to a Slack API (`https://api.slack.com/apis`) instead of a Splunk API. Is that a typo?

Reply:
That is 100% a typo and will be fixed!